src/frontend/efiling-frontend/src/components/page/cso-account/CSOAccount.js

```diff
@@ -27,16 +30,56 @@ export default function CSOAccount({
   const [termsAccepted, acceptTerms] = useState(false);
   const [continueBtnEnabled, setContinueBtnEnabled] = useState(false);
   const [showLoader, setShowLoader] = useState(false);
+  const [emailInput, setEmailInput] = useState({ email: "", confEmail: "" });
+  const [emailInputErrors, setEmailInputErrors] = useState({
+    emailError: "",
+    confEmailError: "",
+  });
+  const isBCeID = isIdentityProviderBCeID();
```

Comment:
maybe change the logic, when the applicationInfo does not have an email prompt the user for a valid email address

Reply:
Done.  Logic is now based on simply whether application.email has data or not.